### PR TITLE
[WIP] Quickstart DNM

### PR DESCRIPTION
**Pull request by zefciu**
Just a peek to see if this is what we want.

### CODE REVIEW

**Comment by vi4m** [comment]
Nice, nice, waiting for mo4r3 ;)

File: doc/quickstart.rst [hunk 1]
```diff
@@ -2,23 +2,96 @@ Quickstart
...
+
+1. Click "Add device"
+
+2. Fill the form with the following data
+
+.. image:: _static/ralph_quick_start_add_assets.png
+    :width: 600px
+
+- **Basic info:**
+
```
Comment (quamilek) [inline doc/quickstart.rst]: Remove empty line, it affects the bad display.
ralph_assets quickstart rst at quickstart zefciu ralph_assets

File: doc/quickstart.rst [hunk 2]
```diff
@@ -2,23 +2,96 @@ Quickstart
...
+    - Price: 2000
+    - Deprecation rate: 25
+    - Invoice date: Enter yesterday's date
+- **Additional info:**
+    - Data center: DC A
+    - Server room: Server room A
+    - Orientation: front
+    - Position: 1
+
+- SN/SNs::
```
Comment (quamilek) [inline doc/quickstart.rst]: This section also displays bad.

File: doc/quickstart.rst [hunk 3]
```diff
@@ -2,23 +2,96 @@ Quickstart
...
+  12345
+  67890
+- Barcode/Barcodes::
```
Comment (quamilek) [inline doc/quickstart.rst]: It is worth to add that you can enter multiple assets at once giving many barcodes/sns.

File: doc/quickstart.rst [hunk 4]
```diff
@@ -2,23 +2,96 @@ Quickstart
...
+following into the ``SN`` field and click "Search" after.:
+
+* ``123``
+* ``"123"``
+* ``"12345"``
+
+Note that you can search for whole value only using quotes or for substrings
+without them.
+
+* Erase ``SN`` field and start typing ``Databases`` into ``Service catalog``
```
Comment (quamilek) [inline doc/quickstart.rst]: Could put some screenshot.

File: doc/quickstart.rst [hunk 5]
```diff
@@ -2,23 +2,96 @@ Quickstart
...
+  field. The select ``Databases`` from the live-search you see. Click "Search"
+
+You can also search be virtually any attribute an asset has.
+
+
+Scanning your asset
+---------------------------
+
+Click ``Core`` and ``Search`` in the menu. Find the container your ralph
```
Comment (quamilek) [inline doc/quickstart.rst]: Could put some screenshot.

File: doc/quickstart.rst [hunk 6]
```diff
@@ -2,23 +2,96 @@ Quickstart
...
+or Addresses set up in ``Addresses`` tab.
 
-There are two types of assets -- devices and parts. A device could be a blade
-server, and a part is a component of this server, for example memory or hard
-disk drive. A part can be assigned to a single device at a time. You can move
-parts from one device to another when you need it.
+Click ``Quick Scan`` in menu and enter ``127.0.0.1``. Keeping all plugins
+selected click ``Scan`` button.
 
+After a while all the plugins will finish its work. Note that most of them will
```
Comment (quamilek) [inline doc/quickstart.rst]: Could put some screenshot.

**Comment by vi4m** [comment]
I created branch https://github.com/allegro/ralph/tree/better-docs to be able to collectively fix documentation.

**Pull request closed without merging**